packages/moonstone/TooltipDecorator/util.js

```diff
@@ -144,12 +157,37 @@ const getPosition = function (tooltipNode, clientNode, arrowAnchor, tooltipDirec
 		}
 	}
 
+	// When tooltip is too wide, shift the tooltip so that the first part of the tooltip is always visible. Does not affect tooltips with `tooltipDirection` of `left` and `right`
+	if (overflow.isOverWide && !(tooltipDirection === 'left' || tooltipDirection === 'right') && arrowAnchor !== 'right') {
+		position.left = rtl ? window.innerWidth - tooltipNode.width : 12;
```

Comment:
This `12` should probably be REM converted and documented as to what the 12 represents. Spotlight-outset? TV Keepout?